### PR TITLE
Update exit modal

### PR DESCRIPTION
**Pull request by keca2**
This PR checks the newly implemented adminContext.endDate field to see if the current date is after the endDate of the event. 

image

If it is, (from homepage) the post event survey modal will open. Upon completion of the survey, user will be directed to the results landing page.

**INSTRUCTIONS TO RENDER:**
2 options : Either change the field from adminContext.endDate to be before today's date for testing, or swap the logic in the homePage.tsx (today < end) on line 75.
image

Also in this PR, updates to the modal render for the post event survey:

image

UPDATE 3/13
image

### CODE REVIEW

**Comment by bailee-crandley** [comment]
For the "Take me to the results" button, does it require that all of the above fields are filled out before proceeding? Also, are you able to click off the modal if you want to see the home page?

**Comment by bailee-crandley** [comment]
Would it be super difficult to add an X close button on the modal? I think that might be an accessibility concern for those who use screen readers to navigate.

File: src/pages/postEventSurvey/postEventSurvey.tsx
Comment (aphamdd) [inline src/pages/postEventSurvey/postEventSurvey.tsx]: ignore this change

File: src/pages/postEventSurvey/postEventSurvey.tsx
Comment (juroc95) [inline src/pages/postEventSurvey/postEventSurvey.tsx:152]: @bailee-crandley is this fine? or do you want to change the wording?

Comment (bailee-crandley) [reply src/pages/postEventSurvey/postEventSurvey.tsx:152]: That's fine for now. I have to go in and change the wording for a few of these questions but I was just going to wait until after this PR is merged. I would change it to "If you used a health app to load your steps data, which one did you use?" but I can make that change when I do the other changes as well.

**Review by bailee-crandley — APPROVED** [review]

**Pull request merged**
